scripts/update-i18n.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+set -e
+
+PWD=$(dirname "${BASH_SOURCE[0]}")
+cd $PWD/..
+
+if [ ! -d .tx ]; then
+    echo "please make sure this script is under the scripts directory!"
+    exit -9
+fi
+
+if [ "$1" != "-v" ]; then
+    QUIET_FLAGS="-silent"
+fi
+
+function regenerate_source() {
+    SEAFILE_PROJECT="seafile-client.pro"
+    rm -f $SEAFILE_PROJECT
+    qmake -project -o $SEAFILE_PROJECT
+    lrelease $SEAFILE_PROJECT $QUIET_FLAGS
+    lupdate $SEAFILE_PROJECT $QUIET_FLAGS
+    rm -f $SEAFILE_PROJECT
+}
```

Comment:
Regarding to #372, how did that fail? It has been working for a very long time (since the beginning when we have i18n support in seafile-applet), and I don't think it could fail without a good reason.